src/makeRobotAccountant.js

```diff
@@ -4,9 +4,24 @@
  *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getFirstCall = (x) => {
+    const getSumWithX = (y) => {
```

Comment:
Also you can return callbacks on this way:

```suggestion
  return (x) => (y) => {
```